### PR TITLE
016/master

### PR DESCRIPTION
**Pull request by raq929**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ By the end of this, students should be able to:
 
 -   Give two reasons why inheritance and Mixins are desirable.
 -   Write a class which inherits from another class.
--   Describe why mixins are preferred to inheritance.
+-   Write a mixin
+-   Describe the difference between inheritance, composition, and mixins
```
Comment (BenGitsCode) [inline README.md:17]: I think your use of an oxford comma here is correct, but the talks header should match this. I opened an issue, so no need to address it in this PR.

**Review by BenGitsCode — COMMENTED** [review]
Looks good. I noticed this is missing frozen string literal comments in several files but I don't think that's critical—maybe it could even be an opportunity to fix it in front of them so they can see linter appeasement in action ;)

**Comment by jrhorn424** [comment]
Is there something blocking this merge @BenGitsCode?

**Comment by jrhorn424** [comment]
Oh I see, a few open issues are asking for my input. Fair enough.

**Review by jrhorn424 — APPROVED** [review]
Impressive. Thanks!

**Comment by jrhorn424** [comment]
Impressive wasn't ironic, it was in reference to your work for the 015 sprint.

Anywho, advice for delivery: Try neglecting questions during the demo portion. Repeat the demo as a "code-along", where they implement what you've written. Only take a few questions. Then, take the most questions and allow the most time for the lab.

The advice on taking questions shouldn't apply to asking them; that is, don't change your normal frequency of CFUs.

**Pull request merged**